src/pages/Home/Home.jsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import { AiFillHome, AiOutlineLogout } from 'react-icons/ai';
+import { MdLabelOutline, MdArchive } from 'react-icons/md';
+import { IoTrashBinOutline } from 'react-icons/io5';
+import { CgProfile } from 'react-icons/cg';
+import './Home.css';
+export const Home = () => {
+  return (
+    <div className="body-center">
+      <nav className="navbar">
+        <div className="logo">
+          <h2>liveNote📚</h2>
+        </div>
+        <div className="right-nav">
+          <ul>
+            <li>Github</li>
+            <li>Twitter</li>
+          </ul>
+        </div>
+      </nav>
+      <main className="note-container">
+        <aside className="sidebar">
```

Comment:
Have your sidebar also as a separate component, as you would be having that in multiple files.